### PR TITLE
update recipe to the new format

### PR DESCRIPTION
**Pull request by ocefpaf**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by ocefpaf** [comment]
@isuruf any idea of what is going on Window here?

**Comment by isuruf** [comment]
https://github.com/conda-forge/lastools-feedstock/pull/4#issuecomment-401608416

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,6 +1,8 @@
 mkdir build
 cd build
 
+set "CXXFLAGS= -MD"
```
Comment (isuruf) [inline recipe/bld.bat]: I'm not sure, but you might need to do the same for CFLAGS

Comment (ocefpaf) [reply recipe/bld.bat]: Thanks. I'll wait for the CIs here. I'm not in a hurry b/c after this one comes `qt` :smile:

**Pull request merged**